src/main/assembly/dist/bin/help.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
I'm really in favour of "mean and lean" whenever possible. This script is obviously only there to provide some alternative way of starting the application in the development environment. However, there is a way to start it that does not include "for development only" customizations of the project.

That said, the trick with `dirname $0` is rather neat. I've seen it in your code before and wondered if we could make it the standard way, as it would obviate the use of the environment variable.